### ISSUE TITLE
Use alpine instead of ubuntu 

### ISSUE THREAD

**Issue by fpaupier** [fpaupier/rnnoise#2]
to shrink resulting image size

**Comment by mohitsaxenaknoldus** [comment]
Please assign it to me, I'll do it.

**Comment by fpaupier** [comment]
Hi @mohitsaxenaknoldus,
Sure thing, your input will be appreciated.

**Comment by mohitsaxenaknoldus** [comment]
@fpaupier PR: https://github.com/fpaupier/rnnoise/pull/4

**Comment by fpaupier** [comment]
Thanks, @mohitsaxenaknoldus. I see you rollbacked to a previous Ubuntu version, does this really provide a significant reduction in the output image size.

**Comment by mohitsaxenaknoldus** [comment]
> Thanks, @mohitsaxenaknoldus. I see you rollbacked to a previous Ubuntu version, does this really provide a significant reduction in the output image size.

Yes, the official `18.04` image is the minimal image but not the previous `20.04` image so it is 50% smaller in size and 40% faster.
Source: https://ubuntu.com/blog/minimal-ubuntu-released
